R/utils.R

```diff
@@ -188,7 +188,9 @@ which_carpentry <- function(carpentry) {
     swc = "Software Carpentry",
     cp = "The Carpentries",
     incubator = "Carpentries Incubator",
-    lab = "Carpentries Lab"
+    lab = "Carpentries Lab",
+    # Default: match the input
+    carpentry
```

Comment:
Coming back to this, I notice that the output of `which_carpentry` is used to generate the alternative text on the logo. If the value set for "carpentry" in `config.yaml` is not the full name of the project/organisation, this would result in less descriptive alt text for custom logos.

@ErinBecker let's discuss this before making a decision.

Reply:
Oh interesting, hadn't noticed that. Could think of another way of these descriptions to be picked up, something that would be more permissive to customisation?
Maybe simply adding a `description` field in the `config.yaml` of a lesson in addition to the `carpentry` field?

Reply:
I was wondering if the metadata handling could be adjusted such that you could specify a value for "carpentry_name" in `config.yaml`, which would be passed through to varnish, with the current behaviour kept as the default for lessons whose `config.yaml` does not contain that field.

E.g. if the config file contains

```yaml
carpentry: 'ice-cream'
carpentry_name: 'Ice Cream Carpentry'
```

'Ice Cream Carpentry' would be used as alt-text on the logo, while

```yaml
carpentry: 'incubator'
# no carpentry_name specified here
```

would use "The Carpentries Incubator" as alt-text, fetched from the defaults set in sandpaper. 

Where a custom `carpentry` was specified, but without an accompanying value provided for `carpentry_name`, e.g.

```yaml
carpentry: 'mexican-guitars'
# no carpentry_name specified here either
```

the value of `carpentry` ("mexican-guitars" in this example) would be used, as in your current approach.

Reply:
This looks like a good solution. The only thing I would suggest is `carpentry_name` should be something more distinct from the `carpentry` option, e.g. `carpentry_description` or `carpentry_title`